### PR TITLE
SecretScan: Remove placeholder image and polish errors

### PR DESCRIPTION
**Pull request by Jguer**
**What is this feature?**

- Remove placeholder image
- Ensure only https is used in production
- Improve http client options

### CODE REVIEW

File: pkg/services/serviceaccounts/secretscan/webhook.go [hunk 1]
```diff
@@ -45,7 +66,6 @@ func (wClient *webHookClient) Notify(ctx context.Context,
 	values := map[string]interface{}{
 		"alert_uid":                uuid.NewString(),
 		"title":                    "SecretScan Alert: Grafana Token leaked",
-		"image_url":                "https://images.pexels.com/photos/5119737/pexels-photo-5119737.jpeg?auto=compress&cs=tinysrgb&w=300", //nolint
```
Comment (alexanderzobnin) [inline pkg/services/serviceaccounts/secretscan/webhook.go:48]: Haha, curious why it was here?

Comment (Jguer) [reply pkg/services/serviceaccounts/secretscan/webhook.go:48]: :smile: it's what happens when things go from hackathon project to project

**Review by alexanderzobnin — APPROVED** [review]

File: pkg/services/serviceaccounts/secretscan/webhook.go (hunk 1 above)
Comment (alexanderzobnin) [reply pkg/services/serviceaccounts/secretscan/webhook.go:48]: Ok, nice bird :smiley:

**Review by cinaglia — APPROVED** [review]
LGTM

**Pull request merged**